### ISSUE TITLE
mutils saveAnim() error

### ISSUE THREAD

**Issue by obriencole11** [krathjen/studiolibrary#212]
Hey there, I'm attempting to call the saveAnim() function in studiolibrary\packages\mutils\animation.py and I get this error:

```
#   File "*\studiolibrary\packages\mutils\animation.py", line 123, in saveAnim
#     sequencePath = mutils.createSnapshot(
# AttributeError: 'module' object has no attribute 'createSnapshot' //
```

I can't seem to find a reference to this createSnapshot() function anywhere, is it possible it's been deprecated? This is on the latest version of studiolibrary. Thanks!

**Comment by krathjen** [comment]
Thanks for the updated.

I have a fix for this and I'll upload it soon, however, this function does not get used by the studio library, so it might be removed or changed in the future.

Cheers,

Kurt

**Comment by krathjen** [comment]
This should now be fixed in version 2.4.0.b16.

Feel free to reopen this ticket if you're still having issues.

Thanks,

Kurt
